components/ILIAS/Contact/PRIVACY.md

```diff
@@ -0,0 +1,87 @@
+# Contact Service Privacy
+
+This documentation does not warrant completeness or correctness. Please report any
+missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+
+- The "Contact" component of ILIAS provides a channel for users to contact each other.
+- The component only handles the contact and shares of information via the contact functionality.
+  There are multiple other ways of context related contacts in ILIAS that are not part of this component, like course 
+  member/tutor relations or test correction processes, etc
+- The global activation of the component can be checked with the
+  "Administration > Communication > Contact > Activate 'Contacts'" setting.
+- The personal activation of the component can be toggled with the
+  "Profile and Privacy > Visibility > Allow Contact Requests" setting.
+- Contacts can be requested within member lists, like in courses and groups.
+- Furthermore, when a user profiles is published, it gives all users access to the profile via the "Who is online" tool
+  and, through that, the option to request contact with that profile.
+  - This can be activated in your profile under "Profile and Privacy > Publish Profile > Visible for logged-in Users"
+    setting.
+  - This setting does not allow or prevent the contact request itself.
+- A contact can be used for two actions:
+  - You can add contacts to your mailing lists
+  - You can invite contacts to chatrooms
+- Furthermore, approved contacts are shown within the "Who is online?" tool
+- The component is not aware of the data being passed through mails or chats. It does not know anything about
+  the origin (i.e. course) of the content or whether it contains user related data or not.
+
+## Data being stored
+
+- Each contact request is stored, including the ID of the requesting user and the ID of the requested user, as well as a
+  timestamp of the request time. This does not require the consent of both target users.
+- When the requested user actively ignores the request, this data entry is added with an "ignore" flag.
```

Comment:
I would like to specify "actively ignore". Imo "ignore" suggests that no action is necessary, so "actively ignore" sounds like  "actively doing smth that doesn't request an action". Suggestion: When the requested user actively ignores the request (by choosing the option "Ignore contact")...